assets/.htaccess

```diff
@@ -0,0 +1,35 @@
+#
+# Whitelist appropriate assets files.
+# This file is automatically generated via File.allowed_extensions configuration
+# See AssetAdapter::renderTemplate() for reference.
+#
+
+# We disable PHP via several methods
+# Replace the handler with the default plaintext handler
+AddHandler default-handler php phtml php3 php4 php5 inc
+
+<IfModule mod_php5.c>
+    # Turn the PHP engine off
+    php_flag engine off
+</IfModule>
+
+<IfModule mod_rewrite.c>
+    <IfModule mod_env.c>
+        SetEnv HTTP_MOD_REWRITE On
+    </IfModule>
+
+    RewriteEngine On
```

Comment:
These files didn't exist before, do they need to be there?

Reply:
whoops - maybe assets should be ignored?